client/src/components/basic/checkbox/checkbox.module.scss

```diff
@@ -0,0 +1,8 @@
+// TODO: replace font and colors to variables
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+.formControlLabel {
+    font-size: 14px;
+    font-family: "Montserrat", sans-serif;
+    font-weight: 500;
+}
```

Comment:
EOF

Reply:
I am not sure that `font: $base-font-family;` will work, it is the shortcut for weight, size and family. Here you should use     `font-family: $base-font-family;`

Reply:
thank you, fixed